sandbox/docker_entrypoint.sh

```diff
@@ -16,12 +16,20 @@ if [[ $current_user_id == 0 ]]; then
   # Make sure the target user owns everything that Grist needs write access to.
   find $write_dir ! -user "$target_user" -exec chown "$target_user" "{}" +
 
+  # Make a home directory for the target user, in case anything needs to access it.
+  export HOME="/tmp/users/${target_user}"
```

Comment:
Why `/tmp`? I'm not sure of the consequences of that location, as it usually has different semantics. I think it's fine in this case, but sometimes it's periodically cleared out by the OS.

Reply:
Because they shouldn't be using it for anything anyway, in my mind. :D 
It's not a meaningful storage directory, just somewhere they have access to.

I can use `/home` if preferred, pretty easy on this one!

Reply:
If you're satisfied that `/tmp` is safe and good, keep as-is.

Reply:
I'll keep it as-is for now then, since it's not meaningfully more volatile than anywhere else inside the container's filesystem. Easy enough one to modify later if we suddenly find we have a use for the home directory.

Reply:
Actually, changed my mind - might as well make it somewhere persistent, feels like less edge cases can crop up. :rofl: